config/forms/2021_2022.json

```diff
@@ -3611,7 +3611,7 @@
                 }
               ]
             },
-            "underoccupation_benefitcap": {
+            "under_occupation_benefit_cap": {
```

Comment:
Why is this changing? Looks like the original was correct in the General Needs Data Dictionary?

Reply:
This is just changing the URL slug, the data key stays the same.